api/config/cdr_config_local.json

```diff
@@ -26,15 +26,15 @@
       "cdrVersionId": 1,
       "name": "[Removed] Synthetic Dataset v1",
       "archivalStatus": 1,
-      "accessTier": "registered",
       "bigqueryProject": "",
       "bigqueryDataset": "",
       "creationTime": "2018-06-06 00:00:00Z",
       "releaseNumber": 1,
       "numParticipants": 0,
       "cdrDbName": "cdr",
       "hasFitbitData": false,
-      "hasCopeSurveyData": false
+      "hasCopeSurveyData": false,
```

Comment:
These configs are starting to get bloated. We can remove any of the boolean properties from hasFitbitData, hasCopeSurveyData, hasFitbitSleepData and hasSurveyConductData that are false. They are only need to be included when they are true.

Reply:
how about other isDefault set as false? Is that an exception?